frontend/src/app/plans/edit/[id]/page.tsx

```diff
@@ -19,7 +19,7 @@ export default async function CreateNewPlan({ params }: PageProps) {
   }
 
   const facultiesRes = (await fetch(
-    "https://planer.solvro.pl/api/v1/departments",
+    `${process.env.NEXT_PUBLIC_API_URL}/departments`,
```

Comment:
env.mjs

Reply:
jak piszesz jeden blad to nie pisz go znowu bo sie powtarzam, zerknij na swoj kom wyzej

Reply:
możesz zignorować wtedy albo napisać "wyżej", jak raz odpiszesz to jest git